### ISSUE TITLE
Drop quotes for UDF metadata

### ISSUE THREAD

**Issue by xzdandy** [georgia-tech-db/evadb#1010]
### Search before asking

- [X] I have searched the EvaDB [issues](https://github.com/georgia-tech-db/eva/issues) and found no similar feature requests.

### Description

Instead of currently

```
CREATE UDF IF NOT EXISTS PredictHouseRent FROM
( SELECT * FROM HomeRentals )
TYPE Ludwig
'predict' 'rental_price'
'time_limit' 120;
```

We'd like to move to
```
CREATE UDF IF NOT EXISTS PredictHouseRent FROM
( SELECT * FROM HomeRentals )
TYPE Ludwig
PREDICT rental_price
TIME_LIMIT 120;
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by hershd23** [comment]
Hi can I take this up?

**Comment by gaurav274** [comment]
Sure!

**Comment by hershd23** [comment]
Hi regarding this issue. 

I went through files in the parser as well as the final statement_binder evadb/binder/statement_binder.py . I see that predict right now is being added in UDF metadata 
https://github.com/georgia-tech-db/evadb/blob/8aa807bfdcdf72f2b9de2ff60931c65bf7c2ac0e/evadb/parser/lark_visitor/_functions.py#L93
My question is whether I need to move PREDICT to a predicate or let it be in the metadata

**Comment by xzdandy** [comment]
> Hi regarding this issue.
> 
> I went through files in the parser as well as the final statement_binder evadb/binder/statement_binder.py . I see that predict right now is being added in UDF metadata
> 
> https://github.com/georgia-tech-db/evadb/blob/8aa807bfdcdf72f2b9de2ff60931c65bf7c2ac0e/evadb/parser/lark_visitor/_functions.py#L93
> 
> 
> My question is whether I need to move PREDICT to a predicate or let it be in the metadata

Hi Hersh, the `PREDICT` should be in the metadata.

**Comment by xzdandy** [comment]
> 

I saw in your pr you have used `uid` instead of `string_literal`. Is this still a problem?

**Comment by hershd23** [comment]
No no, I just wanted to link the issue to the PR. That's it. Making it a UID solves the problem

**Comment by hershd23** [comment]
Closed as #1026 has been merged into staging
